ci-operator/step-registry/sandboxed-containers-operator/get-kata-rpm/sandboxed-containers-operator-get-kata-rpm-commands.sh

```diff
@@ -31,7 +31,7 @@ if [ $err -ne 0 ]; then
 fi
 
 # checks for a bad URL
-if [ "$(grep -q 'title.*404 Not Found' kata-containers.rpm)" -a "$(grep -q 'p.The requested URL was not found' kata-containers.rpm)" ]; then
+if [ "$(grep -q 'title.*404 Not Found' kata-containers.rpm)" ] && [ "$(grep -q 'p.The requested URL was not found' kata-containers.rpm)" ]; then
```

Comment:
This should be part of the previous commit as it was introduced there (there is no need to push additional commits as, unless you rebase, one can see the diff-from-previous-version)

Reply:
Squashed into 1 commit
Fixed the yaml formatting - vscode was reformatting it automatically